test/server/cards/02_SHD/events/EnforcedLoyalty.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe('Enforced Loyalty', function() {
```

Comment:
Suggestion: Can we also add tests to make sure this resolves and does as much as it can in the event we have 0 cards in deck or 1 card in deck?

Reply:
Yeah for sure. Since we've tested that in a few places already I don't want to do that test for every card that has a draw effect, but in this case since "if you do" is a new thing that may different behaviors around effects resolving correctly I think it's worth testing.